### ISSUE TITLE
"Add `::<>`" should trigger signature help

### ISSUE THREAD

**Issue by jonas-schievink** [rust-lang/rust-analyzer#12031]
```rust
fn f() {
    std::mem::size_of$0();
}
```

The "Add `::<>`" assist turns this into `std::mem::size_of::<$0>()`, but does not trigger the signature help popup.

Unlike completions, assists don't currently have this functionality built in, so some extra work is needed.

**Comment by jonas-schievink** [comment]
It appears that our non-support here is intentional, but I'm not sure why this isn't a problem for completions:

https://github.com/rust-lang/rust-analyzer/blob/e3ec87730aaabccff4200b218528bd5f7fa57ff9/crates/rust-analyzer/src/handlers.rs#L1048-L1053

cc @matklad

**Comment by matklad** [comment]
So, originally CodeActions didn't support `edit` property. To actually apply an edit, code action would ping a server with a command, and the server would than apply the edit. This is the bit we intentionally don't support -- the edit *must* be specified in the action directly. 

But it should be fine to specify an auxiliary command to be applied *after* edit!
